certificate/views.py

```diff
@@ -1135,17 +1127,22 @@ def osdag_workshop_download(request):
                 user = user[0]
         name = user.name
         purpose = user.purpose
-        year = '16'
+        year = user.ws_year[2:]
```

Comment:
Even here the ```user``` is not suitable.
But let it be as it was from start and the model structure and attribute names are such that at this point its difficult to change.

Reply:
ok. I kept it as it is